include/TRFFitter.h

```diff
@@ -3,8 +3,8 @@
 
 #include <vector>
 
-#include "TNamed.h"
-#include "Rtypes.h"
+#include <TNamed.h>
+#include <Rtypes.h>
```

Comment:
And just a picky comment here, Vinzenz spent some time going through the code and making all of the root includes have the quotations. It has to do with the way the compiler looks for the headers. I'd just hate to do this after he spent time hunting them all down everywhere else.